backend/core/package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.2.3",
   "description": "Listings service reference implementation for the Bloom affordable housing system",
   "author": "Marcin Jedras <marcin.jedras@exygy.com>",
-  "private": true,
+  "private": false,
```

Comment:
Is the intent here to start publishing this to NPM? I'm not morally opposed to superseding the current `shared/core` package if we think packaging from here makes more sense (though I do have size concerns we'd need to address with a tight `.npmignore`), but I got the impression for a comment on another issue that we didn't want to do that?

Reply:
Ooops, no! This was actually a mistake, I didn't mean to commit it. I was having trouble getting `ui-components` package to import the `backend-core` package, and I briefly thought this might fix the workspace setup (turns out I just had the `index.js` file in the wrong directory at first). I think this should probably stay `private: true` for now at least.

Reply:
#whywehavecodereview